### ISSUE TITLE
Metrics Display error using SQLite

### ISSUE THREAD

**Issue by f0o** [cachethq/Cachet#1044]
Hi,

I recently installed Cachet using the `1.2` branch.
After adding all my components and such and seeing some first great awesome'ness I decided to write down a nagios-plugin interpreter to populate metrics graphs for some components.

I notice that feeding metric values is working but whenever I want to show the metric on the status page, the page returns a 500 Error from laravel.

Digging into the Logs shows this:
```
Next exception 'ErrorException' with message 'SQLSTATE[HY000]: General error: 1 no such function: to_char [SQL-Statement Here]
``` 

I think this happens because I'm using SQLite.

Anything I can help with?

Cheers,
f0o

//Edit: This seems to be related #921

**Comment by f0o** [comment]
In an attempt to fix it by checking out on `e1ae1a1316d3524158ebf7627252b8cd8cb79ea4` resulted in the following error after running `composer update --no-dev -o`:

`PHP Fatal error:  Class 'McCool\LaravelAutoPresenter\LaravelAutoPresenterServiceProvider' not found in /home/f0o/cachet/bootstrap/cache/compiled.php on line 6892`

Sooo... What am I missing? :)

**Comment by f0o** [comment]
Nevermind, Closing.

Fix was to remove the entire contents of `/bootstrap/cache/` and rerun the composer command.

Metrics work now, Awesome thing!
